src/main/kotlin/org/rust/cargo/project/module/util/RustCrateUtil.kt

```diff
@@ -17,23 +14,17 @@ object RustCrateUtil
 fun Module.getSourceRoots(includingTestRoots: Boolean = false): Collection<VirtualFile> =
     ModuleRootManager.getInstance(this).getSourceRoots(includingTestRoots).toList()
 
-val Module.rootMod: RustModItem?
-    get() = getSourceRoots(includingTestRoots = false).firstOrNull()?.let {
-        var target = when (getOptionValue(Module.ELEMENT_TYPE)) {
-            RustExecutableModuleType.MODULE_TYPE_ID -> RustModules.MAIN_RS
-            RustLibraryModuleType.MODULE_TYPE_ID    -> RustModules.LIB_RS
-            else                                    -> return null
-        }
+val Module.crateRootFiles: Collection<VirtualFile>
```

Comment:
This doesn't seem as a very handy method to be exposed as an API. I'd rather replace extension method with an ordinary one and rather added `isCrateRoot(f: VirtualFile)` method.